test/libsolidity/SemanticTest.cpp

```diff
@@ -518,7 +518,7 @@ TestCase::TestResult SemanticTest::runTest(
 				m_gasCostFailure ? TestFunctionCall::RenderMode::ExpectedValuesActualGas : TestFunctionCall::RenderMode::ActualValuesExpectedGas,
 				_formatted,
 				/* _interactivePrint */ true
-			) << endl;
```

Comment:
Can you add back the `endl` as well - `SemanticTest.cpp` should remain unchanged.

Reply:
I will get it put back now.